app/src/main/res/layout/fragment_media_player.xml

```diff
@@ -1,19 +1,134 @@
 <?xml version="1.0" encoding="utf-8"?>
 <androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
     xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
     android:layout_width="match_parent"
     android:layout_height="match_parent">
 
+    <ImageView

```

Comment:
Move all these children into a vertical Linear Layout and set the layout weight accordingly. Then test it on multiple phones with varying screen sizes. This current layout does not fit on smaller phones.